frontend/pages/admin/IntegrationsPage/IntegrationsPage.tsx

```diff
@@ -345,23 +381,23 @@ const IntegrationsPage = (): JSX.Element => {
           onCancel={toggleAddIntegrationModal}
           onSubmit={onCreateSubmit}
           backendValidators={backendValidators}
-          integrations={integrations || []}
+          integrations={integrations || { jira: [], zendesk: [] }}
           testingConnection={testingConnection}
         />
       )}
       {showDeleteIntegrationModal && (
         <DeleteIntegrationModal
           onCancel={toggleDeleteIntegrationModal}
           onSubmit={onDeleteSubmit}
-          url={integrationEditing?.url || ""}
+          name={integrationEditing?.name || ""}
         />
       )}
-      {showEditIntegrationModal && (
+      {showEditIntegrationModal && integrations && (
```

Comment:
Editing an existing integration is being sunsetted for 4.14. I removed the action button to reach this modal, but left all related code in case we eventually add edit integrations back. Thoughts?